### PR TITLE
chore: link to discourse when choosing issue template

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

**Review by clason — APPROVED** [review]

**Comment by clason** [comment]
Or maybe just "Usage questions should be asked on [Discourse](...) or [Gitter](...)."

**Comment by dundargoc** [comment]
> Or maybe just "Usage questions should be asked on [Discourse](...) or [Gitter](...)."

Markdown isn't enabled so can't hyperlink. The best I can do is write out the url in plaintext. We also have to provide a single url for the button on the right, so we'd have to choose one of them. Your suggestions makes sense, which is unfortunately a disadvantage when it comes to github ;) .

**Comment by dundargoc** [comment]
Maybe just link to discourse and leave out Gitter?

**Comment by clason** [comment]
Yes, let's do that. (Discourse has more persistence, which is what we want here -- avoid people asking the same questions over and over...)

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Question
+    url: https://neovim.discourse.group/
+    about: Please ask and answer questions about Neovim on Discourse.
+  - name: Chat
+    url: https://gitter.im/neovim/neovim
+    about: Join Gitter to chat with fellow Neovim users.
```
Comment (vigoux) [inline .github/ISSUE_TEMPLATE/config.yml:8]: Maybe link to matrix instead of gitter, as those two merged now, we should advertise matrix more.

**Comment by dundargoc** [comment]
I've opted to remove the link to gitter/matrix. Partly because it has more persistence like @clason mentioned but also because in hindsight it's a bit goofy to put in a "Bored? Talk to us!" when looking to report an issue.

**Review by vigoux — APPROVED** [review]

**Pull request merged**